include/simdjson/portability.h

```diff
@@ -48,6 +70,7 @@
 #endif
 
 // under GCC and CLANG, we use these two macros
+#define TARGET_FALLBACK TARGET_REGION("")
```

Comment:
That is a bit odd. Why not define TARGET_FALLBACK as 'nothing'?

Reply:
Because UNTARGET_REGION requires that you push something.

Reply:
> Because UNTARGET_REGION requires that you push something.

But why do we need UNTARGET_REGION in the first place?
What purpose does all of this serve?

```
src/fallback$ grep TARGET *
bitmanipulation.h:TARGET_FALLBACK
bitmanipulation.h:UNTARGET_REGION
implementation.h:TARGET_FALLBACK
implementation.h:UNTARGET_REGION
numberparsing.h:TARGET_FALLBACK
numberparsing.h:UNTARGET_REGION
stage1_find_marks.h:TARGET_FALLBACK
stage1_find_marks.h:UNTARGET_REGION
stage1_find_marks.h:TARGET_FALLBACK
stage1_find_marks.h:UNTARGET_REGION
stage2_build_tape.h:TARGET_FALLBACK
stage2_build_tape.h:UNTARGET_REGION
stringparsing.h:TARGET_FALLBACK
stringparsing.h:UNTARGET_REGION
```

Reply:
(This is nitpicking on my part. I don't really care.)

Reply:
Note that I did not remove these things in my PR because I figured there might be a rationale I am not seeing.

Reply:
I'm fine with removing it, honestly. I did it to make it look and feel the same as the other Intel architectures, but we don't do it on ARM either, so it's fine to remove.